src/svg/qsvggenerator.cpp

```diff
@@ -552,6 +654,30 @@ class QSvgPaintEngine : public QPaintEngine
                       "font-style=\"" << d->attributes.font_style << "\" "
                    << endl;
     }
+
+    // Set the current clipPath element for the defs containing the given path, and the currentClipID
+    void clipPathToSvg(const QPainterPath &clipPath, bool clippingEnabled, const QPen &pen) {
+        Q_D(QSvgPaintEngine);
+
+        d->currentClipString.clear();  // We will always overwrite or blank this
+
+        if (!clippingEnabled || clipPath.isEmpty())      
```

Comment:
worth using curly braces here too for consistency (see like 671 as example)

Reply:
Done